pkg/identityserver/store/membership_store.go

```diff
@@ -294,3 +295,75 @@ func (s *membershipStore) DeleteAccountMembers(ctx context.Context, id *ttnpb.Or
 		AccountID: account.PrimaryKey(),
 	}).Delete(&Membership{}).Error
 }
+
+func (s *membershipStore) FindSingleOwnerMemberships(ctx context.Context, id *ttnpb.OrganizationOrUserIdentifiers) ([]ttnpb.Identifiers, error) {
+	defer trace.StartRegion(ctx, "find single full rights memberships ").End()
+	accountQuery := s.query(ctx, Account{}).
+		Select(`"accounts"."id"`).
+		Where(fmt.Sprintf(`"accounts"."account_type" = '%s' AND "accounts"."uid" = ?`, id.EntityType()), id.IDString()).
+		QueryExpr()
+	userMembershipQuery := s.query(ctx, &Membership{}).
+		Select(`"memberships"."entity_id","memberships"."entity_type"`).
+		Where(`"memberships"."account_id" = (?)`, accountQuery).
+		Where(`"memberships"."rights" = (?) OR "memberships"."rights" = (?) OR "memberships"."rights" = (?) OR "memberships"."rights" = (?)`,
```

Comment:
I'm not sure `= (?)` is the correct way to check this. It's possible that users have the `_ALL` right combined with other rights if they first had the other rights, and then were granted the `_ALL` right.